proto/crm/guests/guest_history.proto

```diff
@@ -6,13 +6,13 @@ option go_package = "crm/guests";
 
 import "crm/guests/guest_indicator.proto";
 import "crm/guests/guest_cancelled_reservation.proto";
-import "crm/guests/guest_completed_stay.proto";
+import "crm/guests/guest_checked_in.proto";
 import "primitive/monetary_amount.proto";
 
 message GuestHistory {
```

Comment:
You might want to call this, `RetrievedGuestHistory` to emphasize that it's something we're getting. Maybe `GuestHistorySummary` or `SummarizedGuestHistory`